lib/cmds/start.ts

```diff
@@ -46,6 +46,7 @@ if (Config.osType() === 'Windows_NT') {
   prog.addOption(Opts[Opt.VERSIONS_IE])
       .addOption(Opts[Opt.IE32])
       .addOption(Opts[Opt.IE])
+      .addOption(Opts[Opt.IE64])
```

Comment:
As per our discussion offline, remove the `IE` and `IE32` flags and default `arch` to `Win32`